package.json

```diff
@@ -3,8 +3,10 @@
   "version": "1.0.0",
   "description": "Streamlined DOM Events for most.js",
   "main": "dist/dom-event.js",
+  "typings": "dist/dom-event.d.ts",
```

Comment:
Let's do only the 0.18.0 version bump first.  We're close on the TS defs for cujojs/most, but I think we can merge this more quickly without trying to include TS here.